src/cloud/linode/init.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -e  # Exit on error
+
+if [ -f /etc/os-release ]; then
+    . /etc/os-release
+fi
+case $ID in
+    ubuntu)
+        ;;
+    *)
+        echo "This script only works on Ubuntu"
+        exit 1
+        ;;
+esac
+
+
+# Install Terraform
+if ! command -v terraform &> /dev/null
+then
+    echo "Terraform not found. Installing..."
+    curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -
+    sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
+    sudo apt-get update && sudo apt-get install terraform
+else
+    echo "Terraform is already installed."
+fi
+
+# Install kubectl
+if ! command -v kubectl &> /dev/null
+then
+    echo "kubectl not found. Installing..."
```

Comment:
kubectl should be installed using the package manager.
https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-using-native-package-management
Same for other dependencies.